modules/ROOT/pages/map-users-roles-teams.adoc

```diff
@@ -7,7 +7,7 @@ endif::[]
 
 You can map users in a federated organization's group to a team or role. Your Anypoint Platform organization must use an external identity provider, such as PingFederate.
 
-After you have mapped them, users in an organization can log in to Anypoint Platform using the same organizational credentials and access permissions that an organization maintains using SAML, OpenID Connect (OIDC), or LDAP. +
+After you have mapped them, users in an organization can sign in to Anypoint Platform using the same organizational credentials and access permissions that an organization maintains using SAML, OpenID Connect (OIDC), or LDAP. +
```

Comment:
```suggestion
After you have mapped them, users in an organization can sign in to Anypoint Platform using the same organizational credentials and access permissions that an organization maintains using SAML, OpenID Connect (OIDC), or LDAP.
+
```

Reply:
Our style is to put the + on a separate line, which gives a bit more space.
https://confluence.internal.salesforce.com/display/MTDT/Formatting+with+Asciidoc#FormattingwithAsciidoc-ImplementListContinuation(akaIndentedParagraphs)